### DISCUSSION TITLE
你好，欢迎来到我的博客 | 东方赞

### DISCUSSION THREAD

**Discussion by giscus** [dongfangzan/dongfangzan#3] [General]
# 你好，欢迎来到我的博客 | 东方赞

Congratulations, you have made your first post!

https://www.dongfangzan.cn/greetings

**Comment by dongfangzan** [comment]
大家好！

**Reply by dongfangzan** [reply]
测试一个回复

**Comment by Marcustar** [comment]
路过。看了 2021 大厂逃离的文章，点个赞。 2025 的中文互联网有你们而精彩 :-) ❤
